examples/Boost-filesystem-shared/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Hunter/config.cmake) #set BUILD_SHARED_LIBS=ON
```

Comment:
CMAKE_CURRENT_SOURCE_DIR will work fine here, however I prefer to use CMAKE_CURRENT_LIST_DIR. See http://cgold.readthedocs.io/en/latest/tutorials/cmake-sources/includes.html#id2